test/rack/cache/entitystore/redis_test.rb

```diff
@@ -31,6 +31,11 @@ def sha_like?
     cache.must_be_kind_of(::Redis)
     cache.id.must_equal("redis://127.0.0.1:6379/0")
 
+    # TODO: Wait for redis-store#79fe6a2db to land in a release
```

Comment:
can you uncomment these tests? i'll merge this PR after releasing `rediss://` support in redis-store, no need to have a "green build" if it's not actually indicating success

Reply:
Updated the constraint and added uncommented tests in the same style as redis-store/redis-store@79fe6a2db added!